part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+                    "type": "number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "accelerationSensitivityOverTemperature": {
+                    "description": "accelerometer sensitivity change over temperature",
+                    "comment": "units of percentage per degree celsius (%/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "axis": {
+                    "description": "number of axes of acceleration measurement",
+                    "type": "number"
+                },
+                "zerogOffset": {
+                    "description": "output of the accelerometer when no acceleration is applied",
+                    "comment": "units of g-force (g)",
+                    "$ref": "definitions.json#/value"
+                },
+                "zerogOffsetOverTemperature": {
+                    "description": "accelerometer zero-g offset change over temperature",
+                    "comment": "units of milli g-force per degree celsius (mg/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "outputType": {
+                    "description": "measurement output type",
+                    "enum": [
+                        "analog",
+                        "digital"
+                    ],
+                    "type": "string"
+                },
+                "outputResolution": {
+                    "description": "output resolution of acceleration measurement",
+                    "comment": "units of bits",
+                    "type": "number"
+                },
+                "interface": {
+                    "description": "interface(s) for communication to host",
+                    "examples": [
+                        "i2c",
+                        "spi",
+                        "analog"
+                    ],
+                    "type": "string"
+                },
+                "bandwidth": {
+                    "description": "bandwidth of acceleration measurement",
+                    "comment": "units of Hz",
+                    "$ref": "definitions.json#/value"
+                },
+                "outputDataRate": {
+                    "description": "Output Data rate (ODR) of a device",
+                    "comment": "units of Hz",
+                    "$ref": "definitions.json#/value"
+                },
+                "rmsNoise": {
+                    "description": "Broadband rms noise of a device",
+                    "comment": "units of milli g-force",
+                    "$ref": "definitions.json#/value"
+                },
+                "spectralNoiseDensity": {
+                    "description": "Spectral noise density of a device",
+                    "comment": "units of milli g-force per square root of Hz (milli-g/rt-Hz)",
+                    "$ref": "definitions.json#/value"
```

Comment:
again, just a number

Reply:
done

Reply:
Looks like this didn't get picked up, I think we want:

"type: "number"

NOT: "$ref": "definitions.json#/value"